src/test/java/workshops/logic/WorkShopTestUtil.java

```diff
@@ -0,0 +1,37 @@
+package workshops.logic;
+
+import workshops.domain.*;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+final class WorkShopTestUtil {
+	private WorkShopTestUtil() {}
+
+	static Account account1 = new Account(AccountType.LO1, "1", BigDecimal.ONE, Currency.PLN);
```

Comment:
Nie tzymaj tyle obiektow jako statyczne zmienne, to nie jest dobrze

Reply:
czyli zatrzymać je w fake data i jak mam test np. shouldReturnUser to wybierać np z listy po indeksie?

Reply:
Albo zrob metode zwracajaca usera i przkazuj tam rozne parametry, ale nie twórz ich jako statyczne zmienne